http-core/src/test/scala/org/apache/pekko/http/impl/model/parser/HttpHeaderSpec.scala

```diff
@@ -19,7 +19,7 @@ import pekko.http.impl.model.parser.HeaderParser.Settings
 import org.scalatest.matchers.{ MatchResult, Matcher }
 import pekko.http.impl.util._
 import pekko.http.scaladsl.model.{ HttpHeader, _ }
-import headers._
+import headers.{ Trailer => `Trailer`, _ }
```

Comment:
Hmm, this is creative use of `. Not sure if I like it, but I guess it can't hurt.